core/src/main/java/org/bitcoinj/core/Block.java

```diff
@@ -572,15 +382,16 @@ private void checkSigOps() throws VerificationException {
         for (Transaction tx : transactions) {
             sigOps += tx.getSigOpCount();
         }
-        if (sigOps > MAX_BLOCK_SIGOPS)
+        if (sigOps > MAX_BLOCK_SIGOPS) {
```

Comment:
This is an unrelated change snuck in. Sorry for bitching, but it makes reviewing harder for big refactorings like this one.